ERCS/erc-8088.md

```diff
@@ -0,0 +1,164 @@
+---
+eip: 8088
```

Comment:
```suggestion
eip: 8067
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.